source/configuration.js

```diff
@@ -0,0 +1,44 @@
+
+if(Meteor.isServer) {
+
+  var getenv = Npm.require('getenv');
+  // Wrapper
+  Space.getenv = getenv;
+
+  Space.configuration = Space.getenv.multi({
+    sysLog: {
+      enabled: ['SPACE_SYSLOG_ENABLED', false, 'bool']
+    },
+    appLog: {
+      enabled: ['SPACE_APPLOG_ENABLED', false, 'bool']
+    }
+  });
+
+  // Pass down the
+  Meteor.settings = {
+    "public": {
+      sysLog: {
+        enabled: Space.configuration.sysLog.enabled
+      },
+      appLog: {
+        enabled: Space.configuration.appLog.enabled
+      }
+    }
+  };
+
+  __meteor_runtime_config__.PUBLIC_SETTINGS = Meteor.settings["public"];
```

Comment:
i guess this is good enough for now but we need to think about a general api for passing configuration from server to client. Because sooner or later other packages will need to add something to that too and then they can't because it's a closed system (more or less). Or at least it would mean spreading this boilerplate code around.